packages/camlp4/camlp4.4.01+system/opam

```diff
@@ -0,0 +1,14 @@
+opam-version: "1.2"
+maintainer: "jeremie@dimino.org"
+homepage: "https://github.com/ocaml/camlp4"
+license: "LGPLv2"
+build: [
+  ["sh" "./check-camlp4.sh"]
+]
+available: [ preinstalled & (ocaml-version < "4.02") ]
```

Comment:
Well, I assume we need this package for all versions of OCaml prior to 4.02. The version `4.01` is misleading though, would `camlp4.embed` or similar be OK for opam? I guess at worst we can use `camlp4.0`.